### PR TITLE
Added Morgan Error logging

### PR DESCRIPTION
**Pull request by DucharmeEthan**

### CODE REVIEW

**Review by dlarocque — COMMENTED** [review]
Works nice and we don't need tests which is fire

Would it be possible to add timestamps to the logs?
```ts
morgan.token('timestamp', () => {
  return new Date().toISOString();
});
```

**Comment by dlarocque** [comment]
Since the tests pass can we close this now? https://github.com/BarbzCodez/Spendr/issues/87

**Review by dlarocque — APPROVED** [review]
image

File: server/Logs/userLog.txt
Comment (BarbzCodez) [inline server/Logs/userLog.txt:1]: Maybe I'm dumb but do we need this file I thought it creates the `userLog.txt` once it starts logging (?)

Comment (dlarocque) [reply server/Logs/userLog.txt:1]: I noticed that if I delete it, it isn't re-created so maybe it needs to be there 🤷 not sure

Comment (DucharmeEthan) [reply server/Logs/userLog.txt:1]: Yeah it needs to be there it got yucky when I was doing without it before

**Review by lysackm — APPROVED** [review]
Lgtm. Did you make sure that 400 errors get logged?

**Pull request merged**